examples/google-esp-integration/with_sdk_v3/server.js

```diff
@@ -22,7 +23,7 @@ app.engine(".html", ejs.__express);
 app.set("view engine", "html");
 
 app.get("/", (req, res) => {
-  res.render("index", { uid2BaseUrl: uid2BaseUrl });
+  res.render("index", { uid2BaseUrl: uid2BaseUrl, uid2JsSdkUrl: uid2JsSdkUrl });
```

Comment:
Called to render login will also need to be passed uid2JsSdkUrl.

Reply:
added to the login call ✔